src/main/java/mate.academy/Main.java

```diff
@@ -4,13 +4,15 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService =
+                (ProductServiceImpl) injector.getInstance(ProductServiceImpl.class);
```

Comment:
i think better use abstraction , not impl 
```suggestion
                (ProductService) injector.getInstance(ProductService.class);
```